release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -587,6 +587,41 @@ For more information, see xref:../operators/operator_sdk/java/osdk-java-quicksta
 [id="ocp-4-11-machine-api"]
 === Machine API
 
+[id="ocp-4-11-mapi-aws-imdsv2"]
+==== Configuration options for the Amazon EC2 Instance Metadata Service
+
+You can now use machine sets to create compute machines that use a specific version of the Amazon EC2 Instance Metadata Service (IMDS). Machine sets can create compute machines that allow the use of both IMDSv1 and IMDSv2 or compute machines that require the use of IMDSv2.
```

Comment:
Suggestion >> Can this be phrased as **Machine sets can create compute machines that allow the use of both IMDSv1 and IMDSv2 or just IMDSv2**